content/en/community/_index.html

```diff
@@ -41,10 +42,28 @@
     <img src="/images/community/kubernetes-community-final-05.jpg" alt="Kubernetes Conference Gallery" style="width:100%;margin-right:0% important" class="desktop">
   </div>
   <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%;margin-bottom:3%" class="mobile">
-
-<a name="conduct"></a>
+<a name="values"></a>
 </div>
 
+<div><a name="values"></a></div>
+<div class="conduct">
+<div class="conducttext">
+<br class="mobile"><br class="mobile">
+<br class="tablet"><br class="tablet">
+<div class="conducttextnobutton" style="margin-bottom:2%"><h1>Community Values</h1>
+The Kubernetes Community values are the keystone to the ongoing success of the project.<br>
+Their principals can be seen reflected in near every aspect or initiative within the Kubernetes project.

```

Comment:
I'm happy to change this wording, it's just what I could come up with for now^^:;;

Reply:
principals -> principles?

Reply:
```suggestion
These principles can already be seen reflected across many aspects or initiative within the Kubernetes project.
```
?

Reply:
+1 to the edits